public/javascripts/Admin/src/Admin.GSVLabelView.js

```diff
@@ -548,6 +553,11 @@ function AdminGSVLabelView(admin, source) {
         self.modalLabelId.html(labelMetadata['label_id']);
         self.modalStreetId.html(labelMetadata['street_edge_id']);
         self.modalRegionId.html(labelMetadata['region_id']);
+        if(labelMetadata['comments'] != null) {
+            self.modalComments.html(labelMetadata['comments'].join("<hr style=\"margin: 2px 0;\">"));
+        } else {
+            self.modalComments.html("None");
```

Comment:
Can you add translations for "None" as well? Then that translation can be used below for `self.modalPrevValidations` as well! The reason we hadn't added translations for it before is that the `self.modalPrevValidations` is only shown to admins, so we weren't worried about it. But everyone will be able to see this, so we should get translations set up for it!